mlc_config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^https://curl.zuul.vexxhost.dev"
+    }
+  ]
+}
```

Comment:
I still hope that the vexxhost.dev related issues (cert expired) are temporary and will be fixed but maybe we need to accept this fact now. The certs expired on March 8, which is now 9 days ago... :cry:

Reply:
BTW, does this file support comments? Can we add an explanation here what the file is for using such?

Reply:
Done here: https://github.com/curl/curl/pull/8597/commits/d7e9292a8f35cff16f4f163f8c728e799110c675